src/util/file-filter.js

```diff
@@ -68,15 +72,8 @@ export class FileFilter {
    *  Resolve relative path to absolute path if sourceDir is setted.
    */
   resolve(file: string): string {
```

Comment:
I guess we can completely drop this method since it is only used by `FileFilter.addToIgnoreList`.